test/helper.rb

```diff
@@ -195,6 +195,11 @@ def md.directory?
       mock.mock_command('', '', '', 0)
     }
 
+    cmd_stderr = lambda { |x|
+      stderr = ::File.read(::File.join(scriptpath, '/unit/mock/cmd/'+x))
```

Comment:
We don't need to do string concatenation or interpolation when we're already in a `File.join` call. It's also customary to remove the leading and trailing slashes and let `File.join` do its thing:

```ruby
stderr = ::File.read(::File.join(scriptpath, 'unit/mock/cmd', x))
```

I also question whether we really need another helper here, especially when this is also doing an `exit 1` like `cmd_exit_1` is already doing. Should we just enhance `cmd_exit_1` to accept an optional path to stderr output?

Reply:
I agree with all those points! Update incoming.

Thanks for your always stellar reviews!